packaging/windows/buildpackage.sh

```diff
@@ -30,19 +30,19 @@ fi
 function makelauncher()
 {
 	sed "s|DISPLAY_NAME|$2|" WindowsLauncher.cs.in | sed "s|MOD_ID|$3|" | sed "s|FAQ_URL|${FAQ_URL}|" > WindowsLauncher.cs
-	mcs -sdk:4.5 WindowsLauncher.cs -warn:4 -codepage:utf8 -warnaserror -out:"$1" -t:winexe ${LAUNCHER_LIBS} -win32icon:"$4"
+	mcs -sdk:4.5 WindowsLauncher.cs -warn:4 -codepage:utf8 -warnaserror -out:"$1" -t:winexe "${LAUNCHER_LIBS}" -win32icon:"$4"
```

Comment:
`${LAUNCHER_LIBS}` is a list of multiple arguments - quoting them here breaks the compilation!

Reply:
I've pushed a fix for this.